introduction.md

```diff
@@ -10,6 +10,8 @@ My deadly skills:
 * Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?